.github/workflows/provision.yml

```diff
@@ -0,0 +1,33 @@
+name: Provision DMX Gatekeeper
```

Comment:
How do we validate that this script is correct after we've generated it?

Reply:
@mimendel that would be testing a 3rd party library - It's like saying how do we validate the entity framework is actually generate the right database - out of scope.